fetch/programmer-humour/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Programmer Humour API</title>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
There is no styles.css file in your project and your styles don't work. Please, fix it.